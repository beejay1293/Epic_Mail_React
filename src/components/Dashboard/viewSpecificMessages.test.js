import React from 'react';
import { mount, shallow, render } from 'enzyme';
import ViewSpecificMessage from './viewSpecificMessage';
import store from '../../store'
describe('view a message', () => {
  it('should render correctly if user wants to view a message', () => {
    const top = shallow(<ViewSpecificMessage store={store}/>);
  
    expect(top).toMatchSnapshot();
  });
});
