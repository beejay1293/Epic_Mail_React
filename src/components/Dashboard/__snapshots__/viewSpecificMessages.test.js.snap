// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`view a message should render correctly if user wants to view a message 1`] = `
<ContextProvider
  value={null}
>
  <viewSpecificMessage
    Dashboardstate="Inbox"
    body="viewMessages"
    dispatch={[Function]}
    errors={Object {}}
    inbox={Array []}
    isAuthenticated={false}
    receiver={Object {}}
    sender={Object {}}
    sent={Array []}
    specificMessage={Object {}}
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
    unread={Array []}
  />
</ContextProvider>
`;
